content/en/docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -149,17 +149,15 @@ Say that your team is deploying an ordinary Rails application. You've run some c
 
 If you're not running Kubernetes or a similar automated system, you might find the following scenario familiar:
 
-<div class="emphasize-box" markdown="1">
-
-1. One instance of your app (a complete machine instance or just a container) goes down.
+{{< note >}}
+1. One instance of your app (a complete machine instance or just a container) goes down.</li>
 
-2. Because your team has monitoring set up, this pages the person on call.
+1. Because your team has monitoring set up, this pages the person on call.</li>
```

Comment:
The inner content of the note shortcode is treated as Markdown. Not sure what you need those end li-tags.